pkg/base/util/drive_helper.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+// IsDriveSystem check if given uuid is in systemDisks array
+// Receive: uuid of drive, array of system disks uuid
+// Return: true if uuid is in systemDisks, otherwise false
+func IsDriveSystem(uuid string, systemDisks []string) bool {
```

Comment:
Why do you place that method in separate file? You have method `isDriveSystem` in volume manager that does real work. But here you just check that slice contains string. We have method that check whether slice contains string or not. Let's use it.